packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -417,6 +417,27 @@ class FileCollection extends DocumentCollection {
     return this.stackClient.fetchJSON('POST', uri`/files/trash/${id}`)
   }
 
+  /**
+   * Copy a file.
+   *
+   * @param {string} id   - The file's id
+   * @param {string} name   - The file copy name
+   * @param {string} dirId   - The destination directory id
```

Comment:
Since those params are optional you can mark them like this: 
```
 * @param {string} [name]   - The file copy name
 * @param {string} [dirId]   - The destination directory id
 ```